community/kernel/src/test/java/org/neo4j/unsafe/batchinsert/internal/BatchInsertTest.java

```diff
@@ -1722,6 +1559,19 @@ private void setAndGet( BatchInserter inserter, Object value )
         return array;
     }
 
+    private Node getNodeInTx( long nodeId, GraphDatabaseService db )
+    {
+        try ( Transaction ignored = db.beginTx() )
+        {
+            return db.getNodeById( nodeId );
+        }
+    }
+
+    private void forceFlush( BatchInserter inserter )
+    {
+        ((BatchInserterImpl)inserter).getNeoStores();
```

Comment:
Alright, the thinking was that you get NeoStores to look at records things and when you do you want all changes to be present. Sure this method could be used to get the NeoStores at after creation and then just use it. I see your point.